app/metrics.yaml

```
@@ -239,6 +240,8 @@ events:
       - android-probes@mozilla.com
       - erichards@mozilla.com
     expires: never
+    no_lint:
+      - BASELINE
```

I see some failures because of

>  Execution failed for task ':app:gleanGenerateMetricsSourceForDebug'.
> Glean code generation failed.
>   
>  Sorry, Glean found some glinter nits:
>  ERROR: BASELINE_PING: events.normal_and_private_uri_count: The baseline ping is Glean-internal. Remove 'baseline' from the send_in_pings array.
>  
>  Please fix the above nits to continue.
> To disable a check, add a `no_lint` parameter with a list of check names to disable.
>  This parameter can appear with each individual metric, or at the top-level to affect the entire file.

```suggestion
      - BASELINE_PING
```